modules/visual_script/doc_classes/VisualScriptIndexGet.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptIndexGet" inherits="VisualScriptNode" version="4.0">
 	<brief_description>
+		A Visual Script node for getting a value from an array or a dictionary.
 	</brief_description>
 	<description>
+		[VisualScriptIndexGet] will return the value stored in an array or a dictionary under the given index.
```

Comment:
[VisualScriptIndexGet] will return the value stored in an array or a dictionary under the given index or key.